packages/api/src/routes/admin.route.ts

```diff
@@ -7,9 +7,9 @@ import { route } from "../util/handlers";
 const router = Router();
 
 router.post(
-  "/promote",
+  "/users/:id",
```

Comment:
what is the full route? POST /admin/users?

Reply:
yes it's POST /admin/users